dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -358,12 +358,12 @@ public String getContent(String mainDescriptorPath, String mainDescriptor, Set<S
                         stepToType.put(workflowStepId, expressionToolType);
                     } else if (run instanceof Map) {
                         // must be import or include
-                        Object importVal = ((Map)run).get("import");
+                        Object importVal = ((Map)run).get("$import");
```

Comment:
Was the `$` optional at some point?

Reply:
I'm not sure. Looking at their documentation, I only see references with  the`$` for include and import

Reply:
Were you going to fetch both, with and without the `$`, or did you decided that with the `$` is the only way to go? If `$` is the only way to go, does that mean `$import` and `$include` have never worked?

Reply:
Originally I was only going to do it with `$` because I didn't see cwl documentation that didn't have it...but now I see this code https://github.com/dockstore/dockstore/pull/3794/files/9f19334385fe1fd251b7189ca3938b378cb411fd#diff-f9e0ac900016e3693ac10f2a3a7b8939L1505 that includes both. I think it is wrong to not have `$`, but maybe safer to just include both for now?

Reply:
I think you have the wrong link, it links back to this comment.

I don't know enough about CWL to know the answer. Maybe you could query all of our CWL source files to see if they're present without the`$`?

Reply:
I think this is the link that was intended to illuminate the import forma: xhttps://www.commonwl.org/v1.0/SchemaSalad.html#Import_example

Reply:
I think Charles has it, if we know that all Dockstore workflows always have the `$` then I would feel comfortable making it mandatory.  The other option is including both for now (essentially because a little more permissive than the current CWL specification). 

I don't have a strong option as to which, but it does feel like this was intentional at some point. So the git history might have a clue if a test or commit comment was included